cmd/ponzu/usage.go

```diff
@@ -162,6 +162,19 @@ var usageVersion = `
 
 `
 
+var usageAdd = `
+[--cli] add, a <addon URI>
```

Comment:
It's not in the end PR, I spotted in late and removed it in [this commit] (https://github.com/ponzu-cms/ponzu/pull/87/commits/5fcca313e9b421a6ed19881bc7d8e28bb0e507bf)